awscli/examples/controltower/update-enabled-baseline.rst

```diff
@@ -0,0 +1,28 @@
+**To Update Control Tower Enabled Baseline**
+
+The following ``update-enabled-baseline`` example updates an AWS Control Tower enabled baseline if baseline 'IdentityCenterBaseline' is **not** enabled::
```

Comment:
```suggestion
The following ``update-enabled-baseline`` example updates an AWS Control Tower enabled baseline if baseline 'IdentityCenterBaseline' is disabled. ::
```
Same as an earlier example, following the API terminology used for enable/disable.